competition/views.py

```diff
@@ -343,8 +344,8 @@ def get_context_data(self, **kwargs):
                     Submission.objects.filter(competitor=OuterRef('pk'), correct=True).order_by(
                         '-problem__level__order').values('problem__level__order')[:1]
                 ),
-                last_correct_submission=Max(
-                    'submission__submitted_at', filter=Q(submission__correct=True))
+                last_correct_submission=(datetime(2022, 11, 8, 0, 0, 0) + (Max(
```

Comment:
Toto tu je take dost neprijemne riesenie. Co to ma robit nevieme to natiahnut z game.,start?